content/authorization/sign-in-with-livechat/index.mdx

```diff
@@ -148,166 +129,210 @@ If they are already logged in, you will immediately receive valuable information
 
 # SDK documentation
 
-## Methods
+## Classes
+
+### **AccountsSDK**
 
-### **accountsSdk.init({ ... })**
+Main instance of SDK using to authorize user in LiveChat Accounts.
 
-> Example `init()` method usage
+#### **constructor(options = {})**
+
+> Example class creation
 
 ```js
-const instance = accountsSdk.init({
-  client_id: "<your_client_id>",
-  onIdentityFetched: (error, data) => {
-    // ...
-  }
+const instance = new AccountsSDK({
+  client_id: '<your_app_client_id>'
 });
 ```
 
-This method initiates the SDK and returns the `accountsSdk` object instance. It accepts an object with the following properties:
+Constructor accepts an `options` object with the following properties:
+
+  - **client_id** - Application registered client ID
+  - **prompt** - use `consent` to force consent prompt in popup and redirect flows, default: `''`
+  - **response_type** - OAuth response type, use `token` or `code`, default: `'token'`
+  - **popup_flow** - `auto` - close popup when not required, `manual` - always show popup, default: `'auto'`
+  - **state** - OAuth state param
+  - **verify_state** - check if state matches after redirect, default: `true`
+  - **scope** - request exact scopes - must be configured for a given client id, string separated with `,`, default: `null`
+  - **redirect_uri - OAuth redirect uri, default: current location
+  - **email_hint** - fill in email in forms
+  - **server_url** - authorization server url
+  - **tracking - tracking querystring params
+  - **transaction** - object with options for transaction manager
+  - **pkce** - object with `PKCE` configuration
+
+Object `transaction` consists from following parameters:
+  - **namespace** - transaction keys prefix, default: `'com.livechat.accounts'`
+  - **key_length** - transaction random state length, default: `32`
+  - **force_local_storage** - try to use local storage instead of cookies, default: `false`
+
+`PKCE` configuration properties:
+  - **enabled** - Oauth 2.1 PKCE extension enabled, default: `true`
+  - **code_verifier** - override auto generated code verifier
+  - **code_verifier_length** - code verifier length, between 43 and 128 characters https://tools.ietf.org/html/rfc7636#section-4.1, default: `128`
+  - **code_challange_method** - code challenge method, use `S256` or `plain`, default: `S256`
 
-- **client_id** – obtained from the [Developers Console](https://developers.livechatinc.com/console/) when you create your app.
+<aside class="notice">
+  Use of PKCE extension is required when JavaScript application wants to acquire authorization code and exchange it for tokens.
+</aside>
 
-- **response_type** – (optional, defaults to `token`) Defines the type of the response that you will receive in `onIdentityFetched` callback. Two options are supported:
+#### **instance.iframe(options = {})** - Returns `Iframe` object instance built on top of sdk.
```

Comment:
Can we keep this format of documenting methods ? 

![image](https://user-images.githubusercontent.com/3074083/108713168-224bf400-7518-11eb-8894-fb67c7118984.png)

Also same below.